### PR TITLE
Update dependencies and run Flow through ESLint

### PR DESCRIPTION
**Pull request by tdeekens**
#### Summary

Updates dependencies and runs Flow through ESLint to get the warnings.

### CODE REVIEW

File: rollup.config.js [hunk 1]
```diff
@@ -4,7 +4,7 @@ const commonjs = require('rollup-plugin-commonjs')
 const json = require('rollup-plugin-json')
 const babel = require('rollup-plugin-babel')
 const replace = require('rollup-plugin-replace')
-const uglify = require('rollup-plugin-uglify')
+const { uglify } = require('rollup-plugin-uglify')
```
Comment (tdeekens) [inline rollup.config.js]: breaking change of the uglify plugin

File: package.json [hunk 2]
```diff
@@ -68,36 +68,36 @@
...
-    "flow-bin": "0.74.0",
+    "flow-bin": "0.75.0",
     "gitbook-cli": "2.3.2",
-    "gitbook-plugin-anchorjs": "1.1.1",
+    "gitbook-plugin-anchorjs": "2.0.0",
     "gitbook-plugin-edit-link": "2.0.2",
-    "gitbook-plugin-ga": "1.0.1",
-    "gitbook-plugin-github": "2.0.0",
+    "gitbook-plugin-ga": "2.0.0",
+    "gitbook-plugin-github": "3.0.0",
```
Comment (tdeekens) [inline package.json]: Maybe generate a gitbook before merging to see that they still work.

**Comment by tdeekens** [comment]
Fails only on CI. Locally for me these updates work. Can you check @daern91?

**Comment by daern91** [comment]
Fails for me locally too.

**Pull request closed without merging**